### ISSUE TITLE
isolate only subtree

### ISSUE THREAD

**Issue by jvanbruegge** [cyclejs/cyclejs#441]
This is a suggestion for the isolate package:
It would be helpful to be able to isolate only siblings from each other (and their respective subtrees).
My particular case is a List of components, all isolated so they dont interfere. Now I want to use DOM.select on the parent to select an element in the isolated children, but currently this is not possible. My only possibility is to make a second component containing the first and the element I want to select and try to use CSS to make them look kinda similar as before.
TL;DR Option to isolate to not block DOM.select from parent

**Comment by staltz** [comment]
This is now done together with issue #526. Check https://cycle.js.org/releases.html#flexible-isolation-cycle-dom

**Comment by jvanbruegge** [comment]
Nice, exactly what I needed
